docs/tpm.md

```diff
@@ -87,7 +88,7 @@ Issue the following command from the terminal:
 	```
 	  Your output should be as follows:
 	```
-	OpenSSL* 1.1.1s  1 Nov 2022
+	OpenSSL* 3.0.7  1 Nov 2022
 	```
 
 #### Steps to install curl version 7.86 configured with openssl
```

Comment:
For the below step, the script has additional parameter specified w.r.t openssl just installed. Should it be updated here ?
./configure --with-openssl --enable-versioned-symbols

Reply:
added